ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceBase.java

```diff
@@ -61,7 +61,7 @@ public final void setup() throws Exception {
 
 		resetPropertiesFile = System
 				.getProperty("com.arjuna.ats.arjuna.common.propertiesFile");
-		if (resetPropertiesFile == null) {
+		if (resetPropertiesFile == null || !resetPropertiesFile.equals("commitmarkableresourcejbossts-properties.xml")) {
```

Comment:
Just to comment to the origin of this property manipulation. The properties file is set to jbossts-properties.xml in the surefire config and as this test needs to set it to commitmarkableresourcejbossts-properties.xml, it caches the original value and then is able to revert it to the value from the surefire config at the end of the test. I can't quite see why the diff added a check for resetPropertiesFile.equals("commitmarkableresourcejbossts-properties.xml") to maintain that  behaviour but perhaps I missed something and if so please can you explain why the proposed change to this line required?